packages/client/src/router/index.js

```diff
@@ -103,6 +110,23 @@ export const routes = [
           }
         },
       },
+      {
+        path: '/my-grants/assigned',
+        name: 'assigned',
+        redirect: { name: 'shared-with-your-team' },
```

Comment:
I think in the last set of changes, you lost the flag check here. We want to only redirect if the new terminology flag is on.

Reply:
This might be a good justification for writing a test for the changes made to this route :)